.github/workflows/fairworkflows.yml

```diff
@@ -6,8 +6,6 @@ name: Python application
 on:
   push:
     branches: [ '**' ]
-  pull_request:
```

Comment:
In pushes to a branch that is merged to master (which is our default workflow), this actually triggers the build job 2 times. Not sure what the rationale for this is, but if it's up to me I would just remove it.

Reply:
Isn't this just the usual CI thing of checking 1) if tests pass on the PR branch, and 2) if they pass after merging? Or perhaps I am misunderstanding what this does